src/main/scala/org/broadinstitute/dsde/workbench/sam/api/UserRoutes.scala

```diff
@@ -93,4 +94,19 @@ trait UserRoutes extends UserInfoDirectives {
       }
     }
 
+  def userPetServiceAccountRoutes: server.Route =
```

Comment:
can we just put this in userRoutes?

Reply:
`userRoutes` is under `/register`, I wanted this under `/api`. See `SamRoutes`:
```
swaggerRoutes ~
    pathPrefix("register") { userRoutes } ~
    pathPrefix("api") { resourceRoutes ~ adminUserRoutes ~ userPetServiceAccountRoutes }
```

Reply:
right... I will think about the naming of that